packages/fe-fpm-writer/test/unit/building-block.test.ts

```diff
@@ -72,10 +72,8 @@ describe('Building Blocks', () => {
         ).toThrowError(/Invalid view path/);
     });
 
-    test('validate sap.fe.core manifest dependency', async () => {
+    test('validate sap.fe.templates and sap.fe.core manifest dependencies are missing', async () => {
         const basePath = join(testAppPath, 'validate-manifest-dep');
-        fs.write(join(basePath, manifestFilePath), JSON.stringify(testManifestContent));
-
         // Test generator without sap.fe.core as dependency in manifest.json
         fs.write(join(basePath, manifestFilePath), JSON.stringify({ ...testManifestContent, 'sap.ui5': {} }));
```

Comment:
can we please add additional test case when we have `sap.fe.templates` and `sap.fe.core` is missing? 
Like:
```
"sap.ui5": {
        "dependencies": {
            "libs": {
                "sap.fe.templates": {}
            }
        }
    }
```

Other tests for building block uses setup:
```
"sap.ui5": {
        "dependencies": {
            "libs": {
                "sap.fe.core": {}
            }
        }
    }
```